test/endpoints/tchannel/baz/baz_metrics_test.go

```diff
@@ -166,30 +125,28 @@ func TestCallMetrics(t *testing.T) {
 		assert.Contains(t, metrics, key, "expected metric: %s", key)
 	}
 
-	inboundLatency = metrics[tally.KeyForPrefixedStringMap(
+	inboundLatency := metrics[tally.KeyForPrefixedStringMap(
 		"test-gateway.test.all-workers.inbound.calls.latency", endpointTags,
 	)]
-	value = *inboundLatency.MetricValue.Timer.I64Value
+	value := *inboundLatency.MetricValue.Timer.I64Value
 	assert.True(t, value > 1000, "expected timer to be >1000 nano seconds")
 	assert.True(t, value < 1000*1000*1000, "expected timer to be <1 second")
 
-	inboundRecvd = metrics[tally.KeyForPrefixedStringMap(
+	inboundRecvd := metrics[tally.KeyForPrefixedStringMap(
 		"test-gateway.test.all-workers.inbound.calls.recvd", endpointTags,
 	)]
 	value = *inboundRecvd.MetricValue.Count.I64Value
 	assert.Equal(t, int64(1), value)
 
-	inboundSuccess = metrics[tally.KeyForPrefixedStringMap(
+	inboundSuccess := metrics[tally.KeyForPrefixedStringMap(
 		"test-gateway.test.all-workers.inbound.calls.success", endpointTags,
 	)]
 	value = *inboundSuccess.MetricValue.Count.I64Value
 	assert.Equal(t, int64(1), value, "expected counter to be 1")
 
 	tchannelOutboundNames := []string{
-		"test-gateway.test.all-workers.tchannel.outbound.calls.latency",
 		"test-gateway.test.all-workers.tchannel.outbound.calls.per-attempt.latency",
 		"test-gateway.test.all-workers.tchannel.outbound.calls.send",
```

Comment:
we do not want `send` either ; out metrics are called `sent` instead of `send` which is why its not in the blacklist yet.

Reply:
This is most likely a typo bug in tchannel: https://github.com/uber/tchannel-go/blob/3abb4c025c1663b383452339a22d918cf9d0be0b/outbound.go#L196

Even though the docs say `outbound.calls.sent`: http://tchannel.readthedocs.io/en/latest/metrics/#call-metrics